### PR TITLE
CCADC-11578 Updated image/GetAction to improve performance.

### PR DESCRIPTION
**Pull request by yeunga**

### CODE REVIEW

File: skaha/src/main/java/org/opencadc/skaha/SkahaAction.java [hunk 1]
```diff
@@ -315,7 +315,6 @@ protected String callHarbor(String idToken, String harborHost, String project, S
         } else {
             harborURL = new URL("https://" + harborHost + "/api/v2.0/projects/" + project + "/repositories/"
                 + repo + "/artifacts?detail=true&with_label=true&page_size=-1");
-            message = "artifacts";
```
Comment (brianmajor) [inline skaha/src/main/java/org/opencadc/skaha/SkahaAction.java]: Why was this removed?

**Review by brianmajor — CHANGES_REQUESTED** [review]
Looks good Alinga, just the one question/comment.

File: skaha/src/main/java/org/opencadc/skaha/SkahaAction.java (hunk 1 above)
Comment (yeunga) [reply skaha/src/main/java/org/opencadc/skaha/SkahaAction.java]: Good eye. It was accidentally removed. Added back.

**Review by brianmajor — APPROVED** [review]
